ReleaseTooling/Sources/ZipBuilder/ZipBuilder.swift

```diff
@@ -452,28 +453,76 @@ struct ZipBuilder {
                                        rootZipDir: zipDir,
                                        builtFrameworks: frameworksToAssemble,
                                        podsToIgnore: analyticsPods)
-
         // Update the README.
         readmeDeps += dependencyString(for: pod.key, in: productDir, frameworks: podFrameworks)
-
-        // Special case for Crashlytics:
-        // Copy additional tools to avoid users from downloading another artifact to upload symbols.
-        let crashlyticsPodName = "FirebaseCrashlytics"
-        if pod.key == crashlyticsPodName {
-          for file in ["upload-symbols", "run"] {
-            let source = pod.value.installedLocation.appendingPathComponent(file)
-
-            let target = zipDir.appendingPathComponent(crashlyticsPodName)
-              .appendingPathComponent(file)
-            do {
-              try FileManager.default.copyItem(at: source, to: target)
-            } catch {
-              fatalError("Error copying Crashlytics tools from \(source) to \(target): \(error)")
+      } catch {
+        fatalError("Could not copy frameworks from \(pod) into the zip file: \(error)")
+      }
+      do {
+        // Update Resources: For the zip distribution, they get pulled from the xcframework to the
+        // top-level product directory. For the Carthage distribution, they propagate to each
+        // individual framework.
+        // TODO: Investigate changing the zip distro to also have Resources in the .frameworks to
+        // enable different platform Resources.
```

Comment:
I thought there was an issue in having them embedded in the frameworks but I can't remember, maybe it was an older Xcode version and latest versions would be fine.

Reply:
We might have had another issue in the past, like an invalid Info.plist. My testing shows it to be fine now.